### PR TITLE
DBT-808 Feat/add tags

### PR DESCRIPTION
**Pull request by kayiganwa04**
## Context

This is a new feature of adding tags

## Test

- As an admin check the sidebar for the Gestion the tag

### CODE REVIEW

File: src/configs/navigation.js [hunk 1]
```diff
@@ -149,6 +149,13 @@ export default {
           to: { name: 'manage-workspaces' },
           roles: 'admin'
         },
+        {
+          icon: 'mdi-tag',
+          key: '',
+          text: 'Gestión de Tags',
+          to: { name: 'manage-tags' },
+          roles: 'admin'
+        },
```
Comment (abelbordonado) [inline src/configs/navigation.js:158]: Remember the comment on the diagram. No need a menu here we just add a button in the workspace list

File: src/modules/resources/components/resources/add-tag-modal.vue [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    return {
+      isOpen: false,
+      loading: false,
+      name: '',
+      tag: ''
+    }
+  },
+  methods: {
+    open(tag) {
+      console.log('tag', tag)
```
Comment (abelbordonado) [inline src/modules/resources/components/resources/add-tag-modal.vue:67]: 🦞

File: src/modules/tags/tags-list/components/tags-list.vue [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+          @emitSearchTextBinding="searchFieldWithDebounce"
+          @emitSearchWord="searchFieldExecuted"
+        />
+      </template>
+
+      <!-- ------------ NO DATA ------------ -->
+      <template v-slot:no-data>
+        <resource-banner-no-data-datatable />
+      </template>
+      <!-- <template v-slot:abel> ABEL </template> -->
```
Comment (abelbordonado) [inline src/modules/tags/tags-list/components/tags-list.vue:39]: Can remove :D

File: src/modules/tags/tags-list/components/tags-list.vue [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            <ResourceButtonAdd text-button="Aggregar" @click="addTag()" />
+            <resource-button
+              icon-button="mdi-autorenew"
+              @click="resetTableOptions"
+            />
+          </template>
+        </Toolbar>
+        <resource-text-field-search
+          :search-word="store.tableOptions.content"
+          label-text-field="Buscar por nombre o DNI o nombre del tag"
```
Comment (abelbordonado) [inline src/modules/tags/tags-list/components/tags-list.vue:29]: 🇪🇸  Buscar por nombre de la etiqueta.

File: src/modules/tags/tags-list/components/tags-table-columns.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+export default [
+    {
+      position: 1,
+      text: 'Nombre',
```
Comment (abelbordonado) [inline src/modules/tags/tags-list/components/tags-table-columns.js:4]: 🇪🇸  Etiqueta

File: src/modules/tags/tags.router.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+const shortcutRoutes = [
+  {
+    path: 'tags',
+    redirect: { name: 'manage-tags' },
+    meta: {
+      middleware: [authMiddleware]
+    }
+  }
+]
```
Comment (abelbordonado) [inline src/modules/tags/tags.router.js:13]: Im not really sure how this shortcut work about

**Comment by abelbordonado** [comment]
Lets get this more aligned

**Comment by abelbordonado** [comment]
Change `Tag` -> Etiqueta
`Tags` -> Etiquetas

File: src/modules/tags/tags.router.js (hunk 6 above)
Comment (kayiganwa04) [reply src/modules/tags/tags.router.js:13]: neither do I but I removed it

**Review by abelbordonado — APPROVED** [review]
Approved! Ready for Customer review

**Pull request merged**